src/allmydata/scripts/startstop_node.py

```diff
@@ -6,27 +6,31 @@
 from allmydata.util.encodingutil import listdir_unicode, quote_output
 
 
-class StartOptions(BasedirOptions):
+class StartStopCommonOptions(BasedirOptions):
+    optParameters = [
+        ["pidfile", None, None, "Tell the node which pid file should use."],
+        ]
```

Comment:
This is kinda confusing indentation for this case. Would pep8/linter be happy if you dedent the outermost closing bracket?

Same in line 22 (and in other files as well).